src/ng/directive/input.js

```diff
@@ -1308,7 +1308,7 @@ function baseInputType(scope, element, attr, ctrl, $sniffer, $browser) {
 
     // if user modifies input value using context menu in IE, we need "paste" and "cut" events to catch it
     if ($sniffer.hasEvent('paste')) {
```

Comment:
Do we have to check for `drop` event support separately? Are there browsers that support one but not the other?

Reply:
I can't imagine that they would support drop but not paste. But perhaps the other way around?

Reply:
From caniuse.com, it looks like support for event drop is equivalent to support for the paste event.